include/MutationOperators/AndOrReplacementMutationOperator.h

```diff
@@ -5,9 +5,13 @@
 #include <vector>
 
 enum AND_OR_MutationType {
-  AND_OR_MutationType_None = 0,
-  AND_OR_MutationType_AND_to_OR = 1,
-  AND_OR_MutationType_OR_to_AND = 2
+  AND_OR_MutationType_None               = 0,
+
+  AND_OR_MutationType_AND_to_OR_Pattern1 = 1,
+  AND_OR_MutationType_AND_to_OR_Pattern2 = 2,
+
+  AND_OR_MutationType_OR_to_AND_Pattern1 = 3,
+  AND_OR_MutationType_OR_to_AND_Pattern2 = 4
 };
```

Comment:
Could you describe those patterns a bit? You are the only holder of this information.